Makefile

```diff
@@ -15,10 +15,10 @@ ifeq ($(SILEO_PLATFORM),iphoneos-arm64)
 ARCH            = arm64
 PLATFORM        = iphoneos
 DEB_ARCH        = iphoneos-arm
-DEB_DEPENDS     = firmware (>= 11.0), firmware (>= 12.2) | org.swift.libswift (>= 5.0), coreutils (>= 8.32-4), dpkg (>= 1.20.0), apt (>= 2.3.0), libzstd1
+DEB_DEPENDS     = firmware (>= 11.0), firmware (>= 12.2) | org.swift.libswift (>= 5.0), coreutils (>= 8.31-1), dpkg (>= 1.19.7-2), apt (>= 1.8.2), libzstd1
```

Comment:
Don’t change depends, as these are actually important for Pro users. Just change them when packing on your end.

Reply:
I thought the goal would not have to be keeping stuff separated as we had discussed earlier, but whatever, I'll make a build option for Elucubratus I guess.

Reply:
Stuff not separated is nice, if it doesn’t come at mine and my users’ expense. Our versioning is different, so our depends have to be different.